.github/workflows/deploy.yml

```diff
@@ -17,6 +17,14 @@ env:
   MAKE_J: 3
 
 jobs:
+  build-bccutil:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/ivanhorvath/ccutil:amazing
```

Comment:
It looks like this container hasn't been updated for 2 years. If anything, wouldn't it be better to use something maintained? Perhaps maintain it ourselves?

Reply:
I was asked by RHD team if it would be possible to run `ccutil` upstream because downstream the build process breaks way too often. This is the official image that is used to build all content, I can send you an internal documentation document on how to set it up if you are interested.

I see no reason not to run it if it is going to be reasonably fast, I just need to update XML metadata files in order to actually build it.

Reply:
I'm not opposed to it, but it would be great if the final commit describes what ccutil is and why we run it.

I'd also give the feedback that they may want to update their container. Perhaps that allows Quay to index the packages and gems which would mean you can actually see which software it contains.

Reply:
Sure will update description, this is a draft. FYI RHD team is actually building a NG build system that should replace ccutil so for this reason there is not any further development of the tool.